### PR TITLE
Busra

### PR DESCRIPTION
**Pull request by BushraChelik**
@cetin-ayaz  hello , is there any change needed? thanks

### CODE REVIEW

**Comment by cetin-ayaz** [comment]
can you see me @BushraChelik

File: src/Room2/New.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package Room2;
+
+public class New {
+
```
Comment (emrah-ilgar) [inline src/Room2/New.java:4]: we can write something

**Review by emrah-ilgar — CHANGES_REQUESTED** [review]

File: src/Room2/New.java (hunk 1 above)
Comment (emrah-ilgar) [reply src/Room2/New.java:4]: i want to add something

File: src/Room2/Room2.java [hunk 2]
```diff
@@ -1,4 +1,7 @@
 package Room2;
 
 public class Room2 {
+    public static void main(String[] args) {
+        System.out.println("hello guys");
```
Comment (emrah-ilgar) [inline src/Room2/Room2.java:6]: please change this part

**Review by emrah-ilgar — CHANGES_REQUESTED** [review]

**Review by emrah-ilgar — APPROVED** [review]
i think it is good

**Review by cetin-ayaz — APPROVED** [review]

**Pull request merged**